src/pages/releases.md

```diff
@@ -49,7 +49,7 @@ MariaDB requires the use of the [mariadb][mariadb-npm] npm package.
 | Sequelize   | [MariaDB][mariadb] | [mariadb (npm)][mariadb-npm] |
 |-------------|--------------------|------------------------------|
 | 7 (alpha)   | >=10.3             | >= 3.0.0 [^⚠️2]              |
-| 6 (current) | >=10.3             | ^2.3.3                       |
+| 6 (current) | >=10.1.44          | ^2.3.3                       |
```

Comment:
We're not testing on 10.1 right now, but would be good to add that

Reply:
We're not, but we should look into that. I think we shouldn't remove support for a database version without doing a major release

Reply:
I'll look into that this week

Reply:
Well, looks like this one needs to be reverted